src/components/itemPage/Item.jsx

```diff
@@ -89,6 +90,11 @@ class Item extends Component {
                     <div className="d-none d-sm-block">{this._getAttributeCreationForm()}</div>
                   </div>
                   {viewpoints}
+                  <div className="Ressources">
+                    <h3 className="h4"><Trans>Ressources</Trans></h3>
```

Comment:
Please use `npm extract` to create the entry in the localization dictionaries and then translate it.